<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadImageMainWindow</class>
 <widget class="QMainWindow" name="LoadImageMainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3000</width>
    <height>3000</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>724</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="vBox">
      <item>
       <widget class="QFrame" name="imageFrame">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hBox">
      <item>
       <widget class="QLabel" name="numberLabelsLabel">
        <property name="text">
         <string># Number of Labels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="numberLabelsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PixelLabel">
        <property name="text">
         <string>Pixel/mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PixelLineEdit">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getLabeledImagePushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Get Labeled Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Image"/>
   <addaction name="actionShow_Original_Image"/>
   <addaction name="actionRotate_90_Counter_Clockwise"/>
   <addaction name="actionRotate_90_Clockwise"/>
   <addaction name="actionFlip_Horizontal"/>
   <addaction name="actionFlip_Vertical"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionRotate_90_Counter_Clockwise">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rotate90A.png</normaloff>:/rotate90A.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 90ยบ Counter Clockwise</string>
   </property>
   <property name="toolTip">
    <string>Rotate 90ยบ clockwise</string>
   </property>
  </action>
  <action name="actionRotate_90_Clockwise">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rotate90H.png</normaloff>:/rotate90H.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 90ยบ Clockwise</string>
   </property>
  </action>
  <action name="actionFlip_Horizontal">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/flipH.png</normaloff>:/flipH.png</iconset>
   </property>
   <property name="text">
    <string>Flip Horizontal</string>
   </property>
   <property name="toolTip">
    <string>Flip Horizontal</string>
   </property>
  </action>
  <action name="actionFlip_Vertical">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/flipV.png</normaloff>:/flipV.png</iconset>
   </property>
   <property name="text">
    <string>Flip Vertical</string>
   </property>
   <property name="toolTip">
    <string>Flip Vertical</string>
   </property>
  </action>
  <action name="actionShow_Original_Image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/reset.png</normaloff>:/reset.png</iconset>
   </property>
   <property name="text">
    <string>Show Original Image</string>
   </property>
   <property name="toolTip">
    <string>Show Original Image</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/document-open.png</normaloff>:/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
   <property name="toolTip">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
